packages/instantsearch.js/src/middlewares/createInsightsMiddleware.ts

```diff
@@ -54,6 +58,7 @@ export function createInsightsMiddleware<
     insightsInitParams,
     onEvent,
     $$internal = false,
+    $$automatic = false,
```

Comment:
This flag is your internal parameter to decide to set `clickAnalytics=true`, which you don't want to do when you know the engine already did it, correct?

If yes, I think it deserves a rename because "automatic" refers to the behavior of auto-sending events, it's not exclusive to Dashboard opt-in IMO.

Reply:
I renamed it `$$clickAnalytics` in [e10f7ea](https://github.com/algolia/instantsearch/pull/5883/commits/e10f7eaab73557f461f36f23ef84b0d9558acbba).